### ISSUE TITLE
liquid_tags plugin for IPython notebooks

### ISSUE THREAD

**Issue by magsol** [jakevdp/PythonicPerambulations#9]
I'm attempting to port your `liquid_tags.notebook` plugin to a new Pelican theme, [Flex](https://github.com/alexandrevicenzi/Flex). I noticed both from the comments in `liquid_tags/notebook.py` and from simply running the plugin with the theme that several CSS tags are colliding. Can you point me to the CSS tags that were "specifically designed to make the results look good within the pelican-octopress theme"?

**Comment by magsol** [comment]
Nevermind; further testing points to all the "scrunched-up" CSS in the auto-generated `_nb_header.html` file--from Twitter Bootstrap in particular--as the source of the clobbering of the CSS. I have no idea how to fix this; any suggestions are appreciated.

**Comment by jakevdp** [comment]
Yep, this is a huge issue, and is the main reason I haven't updated liquid_tags to work with recent IPython versions (in which the CSS scheme is entirely new). It was hours of trial and error to get it working the first time, and I assume that's what it will take in your case as well.

**Comment by magsol** [comment]
A cursory glance over the 4.x documentation says the same thing--the IPython API has changed considerably. I'll let you know how it goes :P
